src/main/kotlin/probation/views.kt

```diff
@@ -29,7 +29,6 @@ fun probationViews(model: Model, views: ViewSet) {
   val ndmis = model.getSoftwareSystemWithName("NDMIS")!!
   views.createSystemContextView(ndmis, "ndmis-context", null).apply {
     addDefaultElements()
-    addNearestNeighbours(model.getSoftwareSystemWithName("Offender Management Allocation Manager")!!)
```

Comment:
No longer required as the `ndmis.uses(omic.allocationManager, ...)` link will be added in the `addDefaultElements` part